app/views/users/new.html.erb

```diff
@@ -1,14 +1,15 @@
 <h1>New User</h1>
 
-<%
-=begin%>
- <%= render 'form', user: @user %> 
-<%
-=end%>
-
-<%
-=begin%>
- <%= link_to 'Back', users_path %> 
-<%
-=end%>
 
+<%= form_with(url: "/users" , method: "post") do %>
+<%= label_tag(:username, "username") %>
+<%= text_field_tag(:username) %>
```

Comment:
- [ ] You should maintain the [previous](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) forms you worked through in the Odin project commented here in this file.